app/js/app/directives/equation_editor/EquationEditor.js

```diff
@@ -3,47 +3,51 @@ define(function(require) {
 
     var MySketch = require("inequality").MySketch;
 
-	return ["$timeout", "$rootScope", "api", function($timeout, $rootScope, api) {
+    return ["$timeout", "$rootScope", "api", function($timeout, $rootScope, api) {
 
-		return {
-			scope: true,
-			restrict: "A",
-			templateUrl: "/partials/equation_editor/equation_editor.html",
-			link: function(scope, element, attrs) {
+        return {
+            scope: true,
+            restrict: "A",
+            templateUrl: "/partials/equation_editor/equation_editor.html",
+            link: function(scope, element, attrs) {
 
                 element.on("touchstart touchmove", "canvas", function(e) {
                     e.preventDefault();
                 });
 
                 var sketch = null;
 
-                scope.canvasOffset = { };
+                scope.canvasOffset = {};
                 scope.draggingNewSymbol = false;
-
                 scope.equationEditorElement = element;
 
                 scope.selectedSymbols = [];
                 scope.selectionHandleFlags = {
                     showCalc: false,
                     showResize: true,
-                    showMove: true
+                    showMove: false
                 };
```

Comment:
The `selectionHandleFlags` are leftovers from a now-unused system, where if you clicked on a symbol you could use a menu to drag it round/change it to a vector/bold it/etc. Are we likely to have that back, and either way do we need these lines?